config/config.go

```diff
@@ -324,6 +348,9 @@ type GlobalConfig struct {
 	HipchatAPIURL    string `yaml:"hipchat_api_url,omitempty" json:"hipchat_api_url,omitempty"`
 	HipchatAuthToken Secret `yaml:"hipchat_auth_token,omitempty" json:"hipchat_auth_token,omitempty"`
 	OpsGenieAPIURL   string `yaml:"opsgenie_api_url,omitempty" json:"opsgenie_api_url,omitempty"`
+	WeChatAPIURL     string `yaml:"wechat_api_url,omitempty" json:"wechat_api_url,omitempty"`
+	WeChatAPISecret  string `yaml:"wechat_api_secret,omitempty" json:"wechat_api_secret,omitempty"`
+	WeChatAPICorpID  string `yaml:"wechat_api_corpid,omitempty" json:"wechat_api_corpid,omitempty"`
```

Comment:
If you use camel case on the naming `CorpID`, you got to separate it with an underscore `corp_id`